Formula/wireshark.rb

```diff
@@ -18,29 +18,32 @@ class Wireshark < Formula
   option "with-gtk+", "Build the wireshark command with gtk+"
   option "with-qt", "Build the wireshark command with Qt (can be used with or without either GTK option)"
   option "with-headers", "Install Wireshark library headers for plug-in development"
+  option "with-nghttp2", "Enable HTTP/2 header dissection"
 
+  depends_on "pkg-config" => :build
   depends_on "cmake" => :build
+  depends_on "c-ares"
+  depends_on "geoip"
   depends_on "glib"
   depends_on "gnutls"
   depends_on "libgcrypt"
-  depends_on "geoip"
-  depends_on "c-ares"
   depends_on "libsmi" => :optional
-  depends_on "lua"
+  depends_on "libssh" => :optional
+  depends_on "lua" => :optional
+  depends_on "nghttp2" => :optional
   depends_on "portaudio" => :optional
   depends_on "qt" => :optional
   depends_on "gtk+3" => :optional
   depends_on "gtk+" => :optional
-  depends_on "libssh"
-  depends_on "nghttp2"
-  depends_on "lz4"
-  depends_on "snappy"
-  depends_on "spandsp"
   depends_on "gnome-icon-theme" if build.with? "gtk+3"
 
   def install
-    args = std_cmake_args
-    args << "-DENABLE_GNUTLS=ON"
+    args = std_cmake_args + %w[
+      -DENABLE_CARES=ON
+      -DENABLE_GEOIP=ON
+      -DENABLE_GNUTLS=ON
+      -DENABLE_SMI=ON
```

Comment:
remove this line (SMI), it is an optional dependency that is handled below.

Reply:
done.